include/Oscillator.h

```diff
@@ -182,7 +184,7 @@ class LMMS_EXPORT Oscillator
 		control.f2 = control.f1 < OscillatorConstants::WAVETABLE_LENGTH - 1 ?
 					control.f1 + 1 :
 					0;
-		control.band = waveTableBandFromFreq(m_freq);
+		control.band = waveTableBandFromFreq(m_freq * m_detuning * Engine::mixer()->processingSampleRate());
```

Comment:
Uh... Why are you multiplying `Engine::mixer()->processingSampleRate()`?

Reply:
I had to actually re-research that, since I already forgot.. :sweat_smile: 

For some reason, the supplied detune values are not simple frequency multipliers, but they are also divided by the current sample rate, as can be seen for example here:

https://github.com/LMMS/lmms/blob/b701e82e3b5981018353ee30905a8e6e9a98e0b5/plugins/triple_oscillator/TripleOscillator.cpp#L175-L180

So this is here just to cancel it out.

Reply:
Okay then.